src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -43,13 +43,17 @@ private void readObject(java.io.ObjectInputStream in)
             doc="Read filters to be disabled before analysis", optional=true, common = true)
     public final List<String> disableFilters = new ArrayList<>();
 
-    @Argument(fullName = "disableAllReadFilters",
-            shortName = "disableAllReadFilters",
-            doc = "Disable all read filters", common = true, optional = true)
-    public boolean disableAllReadFilters = false;
+    @Argument(fullName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
+            shortName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_READ_FILTERS,
+            doc = "Disable all tool default read filters", common = true, optional = true)
+    public boolean disableToolDefaultReadFilters = false;
 
     // Map of read filter (simple) class names to the corresponding discovered plugin instance
     private Map<String, ReadFilter> readFilters = new HashMap<>();
+    // readFilters map is overridden in validateArguments
+    // this set is for keeping the names for the help in alphabetical order
+    // and to use them for checking if an argument exists or not
+    private final Set<String> readFilterNames = new TreeSet<>();
```

Comment:
Do you mean to remove this set or just use a `HashSet` instead? There are two reasons for introducing this field:

* Previously, the help was different if command line validation was before or after the plugin validation. As I point out in my comments, the `readFilters` map is overriden in `validateArguments` for the ones that will be used. But the parsing system also perform custom validation of the tools after the validation of plugin arguments, so if something fails after that is called, the help will be incorrect. In addition, I did not explore other validation paths in Barclay, but it could happen that other validation is included after the plugin one and this may fail again to print all the filters.
* I also used this variable to check for existence of the plugin in the names, because relying in the `readFilters` name seems like a bad idea if the state is not maintained.

I will wait for the next round of comments for modifying this, just to clarify if I should remove it or change it for a hash-based set.